config/boards/imx7sabre.csc

```diff
@@ -0,0 +1,10 @@
+# iMX7d dual core + M4 1Gb RAM SoC 1xmPCIe 2xGBE
+BOARD_NAME="MCIMX7SABRE"
+BOARDFAMILY="mx7dsabre"
+BOOTCONFIG="imx_v7_defconfig"
+MODULES=""
+MODULES_NEXT=""
+#
+KERNEL_TARGET="default"
+CLI_TARGET="buster,bionic:next"
```

Comment:
Remove those definitions since they are deprecated. This way:
https://github.com/armbian/build/blob/master/config/targets.conf